### PR TITLE
Add EBI, adjust Neubias into past tense and add resources

### PR DESCRIPTION
**Pull request by pwalczysko**
- Adding the latest course as event.
- Putting Resources under the Neubias item, putting the Neubias item into past tense

@jburel

### CODE REVIEW

File: events/workshops-ebi-micro-may-2023.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: news-events
+filename: events
+title: May 2023 OME workshops
+main-blurb: The OME Team gave two workshops on OMERO, OME-NGFF and IDR as a part of the Microscopy data analysis course at EMBL-EBI.
```
Comment (jburel) [inline events/workshops-ebi-micro-may-2023.html:5]: I won't use the term workshops here
Practicals will be more suitable to match the terminology used in the course

File: events/workshops-ebi-micro-may-2023.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Duration: 2 hours</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Workshop on Day 4: <a href="https://www.ebi.ac.uk/training/events/microscopy-data-analysis-0/" target="_blank">Distributed/Cloud computing - practical</a>
```
Comment (jburel) [inline events/workshops-ebi-micro-may-2023.html:35]:
```suggestion
                <h2 class="event-day">Practical on Day 5: <a href="https://www.ebi.ac.uk/training/events/microscopy-data-analysis-0/" target="_blank">Distributed/Cloud computing - practical</a>
```

File: events/workshops-ebi-micro-may-2023.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: held remotely
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Workshop on Day 4: <a href="https://www.ebi.ac.uk/training/events/microscopy-data-analysis-0/" target="_blank">Image data resource – practical</a>
```
Comment (jburel) [inline events/workshops-ebi-micro-may-2023.html:20]:
```suggestion
                <h2 class="event-day">Practical on Day 4: <a href="https://www.ebi.ac.uk/training/events/microscopy-data-analysis-0/" target="_blank">Image data resource – practical</a>
```

File: events/workshops-ebi-micro-may-2023.html (hunk 1 above)
Comment (jburel) [inline events/workshops-ebi-micro-may-2023.html:5]:
```suggestion
main-blurb: The OME Team gave two practical sessions on OMERO, OME-NGFF and IDR as a part of the Microscopy data analysis course at EMBL-EBI.
```

File: events/workshops-ebi-micro-may-2023.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <p class="event-day-description">The usage of Image Data Resource (IDR) was explored and it was demonstrated how to access the IDR data using the Python API, to fetch image data and metadata from IDR study associated with a publication. A workflow involving programmatical harvesting of data from multiple Data resources (Humanmine and IDR) was shown. Further, re-analysis of data using an alternative segmentation tool to the one used by the publication authors was performed. The analytical packages used included Cellpose, StarDist and Cell Profiler. In the second workshop on the following day, the Distributed/Cloud computing workshop introduced an emerging cloud-optimized imaging format OME-NGFF currently used by the BioImage Archive and the Image Data Resource. It was shown how it can be used to access cloud-hosted imaging data and how it facilitates analysis in parallel, how to convert other formats into OME-NGFF and how to upload and download OME-NGFF images to and from S3 storage. The image analysis examples included workflows involving Cellpose and StarDist.
```
Comment (jburel) [inline events/workshops-ebi-micro-may-2023.html:51]: Add hyperlinks

**Comment by pwalczysko** [comment]
fixes pushed

File: events/workshops-ebi-micro-may-2023.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                </p>
+            </div>
+        </div>
+		<hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Resources available:</h2>
+                <p class="event-day-description">The <a href="https://github.com/ome/EMBL-EBI-imaging-course-05-2023" target="_blank">repository with jupyter notebooks and scripts</a> and five selected workflows on WorkflowHub. <br>
```
Comment (jburel) [inline events/workshops-ebi-micro-may-2023.html:59]:
```suggestion
                <p class="event-day-description">The <a href="https://github.com/ome/EMBL-EBI-imaging-course-05-2023" target="_blank">repository with jupyter notebooks and scripts</a> and five selected workflows published on WorkflowHub. <br>
```

**Comment by jburel** [comment]
Ran locally using the ``docker_startup`` script.
Looks fine.
We can merge when the build is green
Overall I think we need to think of a better place on our website for people to find the training material

**Pull request merged**